### ISSUE TITLE
Bad color choice for markdown in pop-ups

### ISSUE THREAD

**Issue by jetteim** [grafana/grafana#53339]
**Steps to reproduce**:
- add panel description containing markdown italic and bold markup
- hover mouse over "i" icon
**Actual Result**:
- bold and italic font color is dark grey in the dark UI theme
- bold and italic font color is light grey in the light UI theme
**Expected Result**
- bold and italic font color is white in the dark UI theme
- bold and italic font color is black in the light UI theme
**Relevant WCAG Criteria:** [#.#.# WCAG Criterion](link to https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0)
1.4.3
**Environment**:
- Grafana version: 9.0.6
- Data source type & version: N/A
- User OS & Browser: MacOS 11.6.5, Chrome 103.0.5060.114
- Others: Screenshots:
<img width="412" alt="image" src="https://user-images.githubusercontent.com/15871907/183058515-94140cfb-676f-4347-bcdb-22c23fd03956.png">
<img width="425" alt="image" src="https://user-images.githubusercontent.com/15871907/183058659-86387c52-6e51-4797-82bc-e29162474735.png">

**Comment by zuchka** [comment]
see this demo on play: https://play.grafana.org/d/eFOp6rzVz/53339?orgId=1

**Comment by jetteim** [comment]
@zuchka Yes, this is exactly what I'm talking about
<img width="317" alt="image" src="https://user-images.githubusercontent.com/15871907/183379313-715d8d4a-5538-4dae-abfb-fb07cb60b0ff.png">

**Comment by joshhunt** [comment]
Thanks for reporting @jetteim! Easy enough to fix, it it'll go out in 9.1 _soon_.
